### DISCUSSION TITLE
VLC cannot open RTMP with URL: "rtsp://localhost:1935/live/01" when I push h264 video to RTSP-Simple-Server?

### DISCUSSION THREAD

**Discussion by rndfw01** [bluenviron/mediamtx#1305] [Questions]
I have an RTMP-Publisher to publish h264 video to RTSP-simple-server with URL : "rtsp://localhost:1935/live/01", RTSP-simple-server server shows Log is publishing. However, when I open the stream with vlc, it doesn't work? I think the problem is in "/live/01"

**Comment by aler9** [comment]
1935 is the RTMP port, not the RTSP one. 8554 is the RTSP port.

[a comment by an automated account is not shown]
